src/structurePrinters/class/MethodDeclarationStructurePrinter.ts

```diff
@@ -28,7 +28,7 @@ export class MethodDeclarationStructurePrinter extends FactoryStructurePrinter<M
         this.printOverloads(writer, structure.name, getOverloadStructures());
         this.printBase(writer, structure.name, structure);
 
-        if (this.options.isAmbient)
+        if (this.options.isAmbient || structure.isAbstract)
```

Comment:
Thanks! Did you add a test for this? Might be worth putting this in a separate pull request though.